# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../boehm-gc/sparc64-ia64.patch
# Copyright (C) 2015 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- gc-7.4./configure.ac
+++ gc-7.4./configure.ac
@@ -578,21 +578,21 @@ case "$host" in
     dnl performance under Irix.
     ;;
  sparc-*-netbsd*)
-    machdep="src/sparc_netbsd_mach_dep.lo"
+    machdep="sparc_netbsd_mach_dep.lo"
     ;;
  sparc*-*-linux* | sparc*-*-openbsd* | sparc64-*-freebsd* | sparc64-*-netbsd*)
-    machdep="src/sparc_mach_dep.lo"
+    machdep="sparc_mach_dep.lo"
     ;;
  sparc-sun-solaris2.3)
-    machdep="src/sparc_mach_dep.lo"
+    machdep="sparc_mach_dep.lo"
     AC_DEFINE(SUNOS53_SHARED_LIB, 1,
               [Define to work around a Solaris 5.3 bug (see dyn_load.c).])
     ;;
  sparc*-sun-solaris2*)
-    machdep="src/sparc_mach_dep.lo"
+    machdep="sparc_mach_dep.lo"
     ;;
  ia64-*-*)
-    machdep="src/ia64_save_regs_in_stack.lo"
+    machdep="ia64_save_regs_in_stack.lo"
     ;;
 esac
 AC_MSG_RESULT($machdep)
